src/_pytest/reports.py

```diff
@@ -406,7 +408,9 @@ def __init__(
         self.__dict__.update(extra)
 
     @property
-    def location(self):
+    def location(  # type:ignore[override]
```

Comment:
Ignoring
```
error: Cannot override writeable attribute with read-only property
```

This was a problem prior to my change -- the type annotation just makes it detectable.

Reply:
seems acceptable, its quite a mess